requirements.txt

```diff
@@ -1,5 +1,5 @@
 WTForms[email]>=3.1,<3.2
-flask>=1.1,<3.1
+flask>=3.0
```

Comment:
We could probably remove this and leave it to Superdesk to define what version of Flask to support

Reply:
Done. It worked well (all tests passed) I just needed to do a minor adjustment to Babel init process.